### DISCUSSION TITLE
1.1.10

### DISCUSSION THREAD

**Discussion by jp7677** [avisi-cloud/structurizr-site-generatr#462] [Announcements]
## What's Changed

* Update to JDK 21 LTS

### Features

* Render PlantUML diagrams in Markdown and AsciiDoc documents by @dirkgroot in https://github.com/avisi-cloud/structurizr-site-generatr/pull/411
* Support for image views by @qtzar and @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/413
* Add function to export the DSL as a JSON file. by @qtzar in https://github.com/avisi-cloud/structurizr-site-generatr/pull/414

### Bugfixes

* Fix math rendering in Markdown by @tchamelot in https://github.com/avisi-cloud/structurizr-site-generatr/pull/444

### Dependency updates

* Update dependency org.jetbrains.kotlinx:kotlinx-html-jvm to v0.11.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/412
* Update eclipse-temurin Docker tag to v21.0.2_13-jre-jammy by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/416
* Update dependency gradle to v8.6 - autoclosed by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/429
* Update dependency org.slf4j:slf4j-simple to v2.0.12 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/430
* Update gradle/gradle-build-action action to v3.1.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/433
* Update dependency net.sourceforge.plantuml:plantuml to v1.2024.3 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/436
* Update dependency org.jetbrains.kotlinx:kotlinx-serialization-json to v1.6.3 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/437
* Update dependency org.jruby:jruby-core to v9.4.6.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/438
* Update dependency org.codehaus.groovy:groovy-jsr223 to v3.0.21 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/445
* Update dependency mermaid to v10.9.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/446
* Update dependency org.eclipse.jetty.websocket:jetty-websocket-jetty-server to v12.0.7 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/450
* Update dependency org.eclipse.jetty:jetty-server to v12.0.7 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/451
* Update dependency org.jetbrains.kotlin:kotlin-scripting-jsr223 to v1.9.23 - autoclosed by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/452
* Update dependency org.asciidoctor:asciidoctorj to v2.5.12 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/454
* Update plugin org.jetbrains.kotlin.jvm to v1.9.23 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/455
* Update plugin org.jetbrains.kotlin.plugin.serialization to v1.9.23 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/456
* Update dependency org.eclipse.jgit:org.eclipse.jgit to v6.9.0.202403050737-r by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/457

## New Contributors
* @tchamelot made their first contribution in https://github.com/avisi-cloud/structurizr-site-generatr/pull/444

**Full Changelog**: https://github.com/avisi-cloud/structurizr-site-generatr/compare/1.1.8...1.1.10

<hr /><em>This discussion was created from the release <a href='https://github.com/avisi-cloud/structurizr-site-generatr/releases/tag/1.1.10'>1.1.10</a>.</em>

**Comment by jp7677** [comment]
@dirkgroot Could you perform the Homebrew magic please?

**Reply by dirkgroot** [reply]
Done! ✅

**Reply by dirkgroot** [reply]
And thanks for creating the release!
